src/parser.h

```diff
@@ -162,6 +162,9 @@ const struct lys_node *lyp_get_yang_data_template(const struct lys_module *modul
 
 void lyp_ext_instance_rm(struct ly_ctx *ctx, struct lys_ext_instance ***ext, uint8_t *size, uint8_t index);
 
+int lyp_get_ext_list(struct ly_ctx *ctx, void *elem, LYEXT_PAR elem_type,
```

Comment:
Please add doxygen description with the short comment about the purpose of the function and its parameters.